api/v1alpha1/nats_status_test.go

```diff
@@ -307,6 +342,14 @@ func Test_SetWaitingStateForStatefulSet(t *testing.T) {
 			LastTransitionTime: currentTime,
 		}
 
+		expectedAvailabilityZonesCondition := &kmetav1.Condition{
+			Type:               string(ConditionAvailabilityZones),
+			Status:             kmetav1.ConditionFalse,
+			Reason:             string(ConditionReasonStatefulSetPending),
```

Comment:
Is `ConditionReasonStatefulSetPending` the right condition to use here?

Reply:
The condition is in pending state because that stateful set is not rolled out completely.